doc/changelog/04-tactics/15661-master+simpl-stop-distinguishing-projection-status.rst

```diff
@@ -0,0 +1,6 @@
+- **Fixed:**
+  :tacn:`simpl` not unfolding a primitive projection when the `never`
+  modifier is set for this projection
```

Comment:
```suggestion
  :tacn:`simpl` and :tacn:`cbn` will now not unfold a primitive projection when the `never`
  modifier is set for the projection
```

Accurate?  Not for `cbv` or any others?

Reply:
In full, the modifier is indeed `simpl never`, yes. Not for `cbv` and others, yes.

There is a little inaccuracy but already in the original message: there were cases where the unfolding was already correctly blocked. The PR fixes a situation where it was still not the case.

A completely different message could be:
```
Some cases where :tacn:`simpl` and :tacn:`cbn` did not simplify a primitive projection even though the `simpl never`
  modifier was set for the projection
```

Reply:
How about this tweak of your wording?

```
Some cases where :tacn:`simpl` and :tacn:`cbn` simplified a primitive projection even though the `simpl never`
  modifier (see :cmd:`Arguments`) was set for the projection
```

"did not simplify" -> "simplify"

Reply:
Yes, perfect.